From 8ebf4a3919bcbd70bfc5cddf365ac4c928e7a5a3 Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Fri, 5 Jan 2024 15:17:20 +0100
Subject: [PATCH] Revert "progress: Clear source after reported download
 progress"

This reverts commit d116870e943de2af088e09ccb8cce71aac7be2eb.
---
 core/progress_thread.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/progress_thread.c b/core/progress_thread.c
index 90687f2..161d2f6 100644
--- a/core/progress_thread.c
+++ b/core/progress_thread.c
@@ -24,7 +24,6 @@
 
 #include "swupdate.h"
 #include <handler.h>
-#include "swupdate_status.h"
 #include "util.h"
 #include "pctl.h"
 #include "network_ipc.h"
@@ -122,7 +121,6 @@ static void _swupdate_download_update(unsigned int perc, unsigned long long tota
 		pprog->msg.dwl_percent = perc;
 		pprog->msg.dwl_bytes = totalbytes;
 		send_progress_msg();
-		pprog->msg.source = SOURCE_UNKNOWN;
 	}
 	pthread_mutex_unlock(&pprog->lock);
 }
-- 
2.43.0

